### ISSUE TITLE
ENH: Display location of SNP cluster somwhere in web

### ISSUE THREAD

**Issue by corneliusroemer** [nextstrain/nextclade#706]
I have a sequence that's reported to have 11 mutations in a cluster, but I can't figure out where this cluster is supposed to be.

It would be helpful in these situations if the location of the cluster was displayed somewhere, possibly in the QC tooltip.

![image](https://user-images.githubusercontent.com/25161793/151199231-09a825ed-a05c-439f-b9dc-a17c1fad0c05.png)

**Comment by corneliusroemer** [comment]
Or is this a bug? I can't possibly find a cluster with width 100 and 11(!) private muts there, even a window of 1000 wouldn't be enough.

![image](https://user-images.githubusercontent.com/25161793/151199829-4d3bbc0f-7ebf-44ed-9dae-fcf2814c0e09.png)

**Comment by corneliusroemer** [comment]
I think I've figured it out: private deletion reversions are counted as individual private mutations. So if you have a stretch of 9 deletions plus 2 other private muts you get to 11.

I _think_ this is a bug, possibly because of new private mut feature. Will open as new issue.

**Comment by ivan-aksamentov** [comment]
@corneliusroemer Is this issue the same as https://github.com/nextstrain/nextclade/issues/707? Does it require any work additional to https://github.com/nextstrain/nextclade/pull/710?

**Comment by corneliusroemer** [comment]
Yes, sorry, I've hidden the off topic comments that spawned the new issue.
